libs/service-portal/licenses/src/utils/dataMapper.ts

```diff
@@ -430,3 +433,100 @@ export const mapCategory = (id: string): Category => {
       }
   }
 }
+
+export const getLicenseDetailHeading = (type: GenericLicenseType) => {
+  switch (type) {
+    case GenericLicenseType.DriversLicense:
+      return {
+        title: m.yourDrivingLicense,
+        text: m.drivingLicenseDescription,
+      }
+      break
+    case GenericLicenseType.AdrLicense:
+      return { title: m.yourADRLicense, text: m.adrLicenseDescription }
+      break
+    case GenericLicenseType.MachineLicense:
+      return {
+        title: m.yourMachineLicense,
+        text: m.machineLicenseDescription,
+      }
+      break
+    default:
+      return {
+        title: m.yourDrivingLicense,
+        text: m.drivingLicenseDescription,
+      }
```

Comment:
Does this make sense as a fallback value? Shouldn't it just be "skírteini" or something?

Reply:
Yes done ✔️